next.config.js

```diff
@@ -1,4 +1,13 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+  output: "standalone",
+  webpack: config => ({
```

Comment:
this configuration lets the container watch for code changes and refresh accordingly.